workspaces/announcements/plugins/announcements/src/components/AnnouncementForm/AnnouncementForm.tsx

```diff
@@ -60,10 +60,15 @@ export const AnnouncementForm = ({
     ? DateTime.fromISO(initialData.start_at).toISODate()
     : DateTime.now().toISODate();
 
+  const formattedUntilDate = initialData.until_date
+    ? DateTime.fromISO(initialData.until_date).toISODate()
+    : DateTime.now().plus({ days: 7 }).toISODate();
```

Comment:
Since we are only selecting a date from the UI and not a time, shouldn't this be "today at the end of the day + 7 days" instead of "now + 7 days"? Otherwise, depending on the time when the form is completed, we would be excluding part of the last day.